multiplatform-movies/common/src/commonMain/kotlin/com.novoda.movies.gallery/Gallery.kt

```diff
@@ -1,7 +1,5 @@
 package com.novoda.movies.gallery
 
-import io.ktor.http.Url
-
 data class Gallery(val moviePosters: List<MoviePoster>)
 
-data class MoviePoster(val movieId: Long, val thumbnailUrl: Url)
+data class MoviePoster(val movieId: Long, val thumbnailUrl: String)
```

Comment:
using a primitive here is a bit codesmell - I see you append it later, in which case this is not a thumbnailUrl but a thumbnailUrlFragment, or suffix or something.

Reply:
It is the full url, the fragmet is the one in the `com.novoda.movies.gallery/ApiGallery.kt:12`